index.html

```diff
@@ -0,0 +1,362 @@
+              <li class="bold">Canopy</li>
+              <li class="developed marked">Backend development</li>
+              <li class="year marked">2018</li>
+            </ul>
+          </div>
+
+          <div class="description">
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+          </div>
+
+          <div class="tags">
+            <ul>
+              <li>html</li>
+              <li>css</li>
+              <li>javascript</li>
+            </ul>
+          </div>
+
+          <div class="seeproject">
+            <button type="button" class="see-project-btn">See Project</button>
+          </div>
+        </div>
+        <!-- single work display section ends here -->
+      </section>
+      <!-- section 2 ends Here -->
+
+      <!-- section 3 starts here -->
+      <section class="section3">
+        <div class="about">
+          <h1>About Myself</h1>
+          <p>
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+
+          <div class="social">LET’S CONNECT</div>
+
+          <div class="icons">
+            <ul>
+              <li>
+                <a href="#"><i class="fa fa-facebook fa-2x"></i></a>
+              </li>
+              <li>
+                <a href="#"><i class="fa fa-twitter fa-2x"></i></a>
+              </li>
+              <li>
+                <a href="#"><i class="fa fa-instagram fa-2x"></i></a>
+              </li>
+
+              <li>
+                <a href="#"><i class="fa fa-github fa-2x"></i></a>
+              </li>
+
+              <li>
+                <a href="#"><i class="fa fa-linkedin fa-2x"></i></a>
+              </li>
+            </ul>
+          </div>
+        </div>
+
+        <div class="get-resume">
+          <button type="button" class="get-resume-btn">Get My Resume</button>
+        </div>
+
+        <div class="accordation">
+          <div class="title">
+            <h1>Languages</h1>
+            <div class="icon">
+              <i class="fa fa-angle-down" aria-hidden="true"></i>
+            </div>
+          </div>
+          <div class="floating-elements">
+            <div class="single-elemnt">
+              <img src="image/language/js.png" />
+              <h2>Javascript</h2>
+            </div>
+          </div>
+          <div class="floating-elements">
+            <div class="single-elemnt">
+              <img src="image/language/html.webp" />
+              <h2>HTML</h2>
+            </div>
+          </div>
+          <div class="floating-elements">
+            <div class="single-elemnt">
+              <img src="image/language/css.webp" />
+              <h2>CSS</h2>
+            </div>
+          </div>
+        </div>
```

Comment:
As stated in the project requirements, kindly use ul and li tags for the categories in `About Myself` section. Please make sure that the `Languages`, `Frameworks`, and `Skills` all use the ul and li tags. (check bullet point number 4)

![Screenshot from 2022-09-23 12-45-52](https://user-images.githubusercontent.com/65233860/191992447-7cf671f3-1141-4634-99c3-794f058385ed.png)